src/components/measureEditor/MeasureEditor.tsx

```diff
@@ -306,13 +332,22 @@ const MeasureEditor = () => {
                 buttonSize="md"
                 onClick={() => updateMeasureCql()}
                 data-testid="save-cql-btn"
+                disabled={areLike}
```

Comment:
not being picky here, but the variable "areLike" isn't helping in reading the code. Can we rename it to something which is self-explanatory? Something like "isCqlUpdated" or isCqlUnChanged" or something like that.